addons/website/static/src/scss/website.scss

```diff
@@ -774,3 +786,36 @@ table.table_desc tr td {
         background-repeat: no-repeat;
     }
 }
+
+// Scroll down button
+.o_scroll_button {
+    animation: o-anim-heartbeat 2.6s ease-in-out 1s infinite;
+    width: 50px;
+    height: 50px;
+    display: none;
+
+    &:focus {
+        outline: none;
+    }
+
+    &:hover {
+        text-decoration: none;
+        animation-iteration-count: 1;
+    }
+}
+
+// Hide button when not using a min-height
+.o_full_screen_height, .o_half_screen_height {
+    .o_scroll_button {
+        display: flex;
+    }
+}
```

Comment:
Meh :/ Maybe we can do a hacky better solution for this for now: override updateUIVisibility of the snippet options to check that the UI is visible at the end -> if not: remove the button. (especially since it seems to be a mistake here: the UI is visible only if full screen but the button is also visible in half screen ?)

Reply:
That just sounds like cleanForSave with extra steps.

Reply:
Not really, since the goal it to not need that css rule making it hidden by default and only visible in some case (which you would still need with a cleanForSave).

Reply:
I still need this rule for blog, since the button is always present there. But I can make it specifically for blog and invert the logic:
```
:not(.o_full_screen_height, .o_half_screen_height, .cover_full, .cover_mid) #42522 #o_wblog_post_content_jump{
    display: none;
}
```

I'm not sure what you mean with me still needing that rule with cleanForSave though. I just remove the button from the DOM if it's not visible on save, how is that different from removing it if the UI is not visible "at the end" (and what do you mean by "at the end")

Reply:
What would be best is: the button is in the DOM if it must be visible and the button is not in the DOM if is must not. So:
- When we enable the button we add it in the DOM
- When we disable the button we remove it from the DOM
- When we change the height of the snippets, the button option is hidden/shown -> if it is hidden, the button is removed from the DOM -> if you do not do that on visibility update, then you will need a css rule to hide it temporarily until cleanForSave is called... which is what I want to avoid

For blog, there is no problem: the arrow is in the DOM, so it is visible, always. And since the button option does not exist there, it won't be removed/added in any way there.

Reply:
I don't mind always showing the arrow on the blog, that's just not how it works at the moment, and it looks quite weird when the height is set to fit-content and you click the arrow, because oftentimes the button will scroll into the small zone where the navbar is hidden (below the regular navbar but above the height where the navbar fixed is enabled)

Reply:
Ok there is something I don't understand ;) When I test in master I always have the arrow on blog even in non full/half height. If there is a spec I don't know about which says there should not be the arrow in blog in that case, it is probably a matter of t-if in the xml ?

Reply:
It works that way in 13.0 and the css rules that implement that behaviour have not been changed in master, so if it doesn't work that way in master it's probably a bug. The coverProperties seem broken in master (since before the imagepicker was merged). I didn't have this issue while working on the imagepicker probably because I didn't rebase that branch on master in the last few days. I will investigate and check with @sbuhl but I'm pretty sure that the button is not supposed to show up in fit-content cover mode, and that's currently done in css, although I don't mind changing that to a t-if, since the button is invisible in edit mode anyway and it doesn't make sense to have it in the DOM when it's invisible, especially since we know that at render-time.

Reply:
:+1: Yep, so probably a bug to solve in 13.1/master then the css rule to change into an xml condition.

Reply:
https://github.com/odoo/odoo/pull/42567